modules/monitoring/application/views/scripts/list/servicegrid-flipped.phtml

```diff
@@ -10,6 +10,26 @@ if (! $this->compact): ?>
         <?= $this->sortBox ?>
     </div>
     <?= $this->filterEditor ?>
+    <?php
+    $toggleFilter = Url::fromRequest();
+    if ($toggleFilter->getParam('problems')) {
+        $toggleFilter->remove('problems');
+    } else {
+        $toggleFilter->setParam('problems');
+    }
+    ?>
+  <label class="relevane-filter">
```

Comment:
This should be called `relevanCe`, right? There are other locations too.

Reply:
That is most definitely a typo!
Thanks for pointing it out!